engine/lib/phases/phase_bundle_cycles.mli

```diff
@@ -0,0 +1,6 @@
+(** This phase makes sure the items don't yield any cycle,
+namespace-wise. It makes the namespaces a tree. It does so by creating
```

Comment:
It can be a DAG, not necessarily a tree.

Reply:
thanks, pushing a fix! feel free to change whatever I wrote there, btw!